lib/seattleflu/id3c/__version__.py

```diff
@@ -0,0 +1,7 @@
+"""
+Defines the version of the Seattle Flu ID3C customizations as ``__version__``.
+
+The version scheme is the same as ID3C for the same reasons.  See
+``lib/id3c/__version__.py``.
+"""
+__version__ = '2019.1'
```

Comment:
Does this mean every time we update the version of id3c we also have to update the id3c-customizations version (and vice-versa)?

Reply:
Good question! The answer is no. These two projects are setup to use the same versioning scheme, but the versions are independent. We'll bump 2019.1 to 2019.2 when make a new release/deploy. This will be part of the release/deploy automation yet to come.